### DISCUSSION TITLE
Hardened Fist

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#35] [Card Balance]
EQUIPMENT, FACTION 
 Warrior is a SLAYER during [F] COMBAT.

Suggestion:
EQUIPMENT, FACTION 
 Warrior is a SLAYER during F COMBAT.
1 Action: Double your base F until end of the turn.

**Comment by 21-ric** [comment]
You right, this card needs an update. 😉

![image](https://user-images.githubusercontent.com/83040170/119049720-2d041100-b9c1-11eb-9bb5-7dd5b791da2c.png)

**Reply by Sol-au-fein** [reply]
I like original, but it would make SSW obsolete. You can fight combat only as two actions? Auto-kill without doubling opponents A?

**Reply by Kite80** [reply]
That was the old slayer ability, with new rules you still need to overtake opponent's doubled A

**Reply by Sol-au-fein** [reply]
So you agree with my suggestion?

**Reply by 21-ric** [reply]
For 2 actions you gain 1 tactical action. This warrior double is base [F] and is a SLAYER (FIGHT only) for this combat.
Click on this card to activate this ability (like "mission statement")

**Reply by Sol-au-fein** [reply]
Thats very nice, but way too powerful fór common card. If it is extra tactical action i would remove Slayer then

**Reply by Sol-au-fein** [reply]
Lets think again about that. We agree on doubling F, but under what circumstances?

**Reply by 21-ric** [reply]
EQUIPMENT, FACTION
1 Action: Owning Warrior's base [F] rating is doubled until the begining of your next Draw step.

I agree to remove SLAYER. 

For 1 action :
chasseur +1 to [F]
combat oversear +4 to [F]
coral beach +3 to [F]
cybercurity MP +3to [F]
Dr Diana +3 to [F]
Mirrorman +1 to [F]
Vac engineer +3 to [F]
Vince Diamond +5 to [F]

I think we are close.

**Reply by Sol-au-fein** [reply]
Doubling F makes sense only for Vince with F augments and for Overseer with Fragment of the True Chip, which is not bad. Maybe for Coral Beach after some update and for Cyril Dent in future.

**Reply by 21-ric** [reply]
For 2 action :
chasseur +2 to [F]
combat oversear +8 to [F]
coral beach +6 to [F]
cybercurity MP +6 to [F]
Dr Diana +6 to [F]
Mirrorman + 2 to [F]
Vac engineer +6 to [F]
Vince Diamond +10 to [F]

With "fragment of the true chip" 'combat oversear" will gain +14 to [F] for 2 actions and "chasseur" will gain +8 to [F] for 2 actions.

Or :

EQUIPMENT, FACTION
Owning Warrior's base [F] rating is doubled.

**Reply by Sol-au-fein** [reply]
Fragment doesnt change your base stats, it just add bonus to equal highest base stat im afraid :/

**Reply by 21-ric** [reply]
I'm afraid too. 🙂
The text is : Base [F], [S], and [A] are always equal to the highest number of the three.

**Comment by david-roxygen** [comment]
Could be nice your version. To make things easier in my opinion could be converted in a sort of SSW4200P like 
![Picture10](https://user-images.githubusercontent.com/88280663/130099532-48839ccd-fadd-4e9f-8a8c-7b4f41190ac1.jpg)

**Reply by 21-ric** [reply]
It's the easier solution and probably our best solution. 🙂
I prefer the original picture for this card. 😇
